addon/-private/properties/alias.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+
+import {
+  getProperty,
+  objectHasProperty
+} from '../helpers';
+
+/**
+ * Returns the value of some other property on the PageObject.
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continue: alias('continueButton.click')
+ * });
+ *
+ * // calls `page.continueButton.click`
+ * page.continue();
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continueText: alias('continueButton.text')
+ * });
+ *
+ * // checks value of `page.continueButton.text`
+ * assert.equal(page.continueText, 'Go ahead!');
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   clickContinueButton: clickable('button.continue'),
+ *   continue: alias('clickContinueButton')
```

Comment:
I don't know why anyone would create an alias in this situation; seems like you would just name `clickContinueButton` as `continue` instead. Is it even worth including this example?